Update Core repo readme with new copy and info about Base and System (#32360)
mui/material-ui: README.md

  <a href="https://mui.com/" rel="noopener" target="_blank"><img width="150" src="https://mui.com/static/logo.svg" alt="MUI logo"></a>
</p>

<h1 align="center">MUI Core</h1>

> <div align="center">

Reviewer: This creates issues

<img width="1059" alt="Screenshot 2022-04-18 at 23 17 31" src="https://user-images.githubusercontent.com/3165635/163879533-94b15257-c35f-4c1c-b001-a0312d4f48a3.png">

- _Material UI_ is a comprehensive library of components that features our implementation of Google's [Material Design](https://material.io/design/introduction/) system.

> - _MUI Base_ is our library of "unstyled" components and low-level hooks. With Base, you have complete control over your app's CSS.

Reviewer 2: ```suggestion
- _MUI Base_ is our library of "unstyled" components and low-level hooks. With Base, you have complete control over your app's CSS while gaining accessibility features support out-of-the-box.
```

My suggestion might be a bit wordy so feel free to improve! I do feel like accessibility is a major offered value in terms of Base and we should probably mention it as much as we can :)

Author: How about something simple like this?

```suggestion
- _MUI Base_ is our library of "unstyled" components and low-level hooks. With Base, you gain complete control over your app's CSS and accessibility features.
```

Reviewer 2: Yup! That's great!

## Installation

### Material UI

Material UI is available as an [npm package](https://www.npmjs.com/package/@mui/material).

> **[Stable channel v5](https://mui.com/)**

Reviewer: I think that we could move this out of Material UI. We have one version, and it's for MUI Core, not Material UI.

</details>

**Note:** `@next` only points to pre-releases.
Use `@latest` for the latest stable release.

### MUI Base

MUI Base is available as an [npm package](https://www.npmjs.com/package/@mui/base).

```sh
// with npm
npm install @mui/base

// with yarn
yarn add @mui/base
```

**Note**: MUI Base is still in alpha.

> We are adding new components on a regular basis.

Reviewer: ?

```suggestion
We are adding new components regularly.
```

> ReactDOM.render(<App />, document.querySelector('#app'));

Reviewer: This is outdated for React 18. I have updated the codesandbox linked in https://mui.com/material-ui/getting-started/usage/#quick-start

```

In the interactive demo below, try changing the code and see how it affects the output.
(Hint: change `variant` to `"outlined"` and `color` to `"secondary"`.
For more options, see the [`Button` component page](https://mui.com/material-ui/react-button/) in our docs.)

[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/4j7m47vlm4)

## Questions

For how-to questions that don't involve making changes to the code base, please use [Stack Overflow](https://stackoverflow.com/questions/tagged/mui) instead of GitHub issues.

> Use the "material-ui" tag on Stack Overflow to make it easier for the community to find your question.

Reviewer: ```suggestion
Use the "mui" tag on Stack Overflow to make it easier for the community to find your question.
```

[<img loading="lazy" alt="GitHub" src="https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png" height="25">](https://github.com/)

GitHub lets us host the Git repository and coordinate contributions.

[<img loading="lazy" alt="Netlify" src="https://cdn.netlify.com/15ecf59b59c9d04b88097c6b5d2c7e8a7d1302d0/1b6d6/img/press/logos/full-logo-light.svg" height="30">](https://www.netlify.com/)

Netlify lets us distribute the documentation.

[<img loading="lazy" alt="CrowdIn" src="https://support.crowdin.com/assets/logos/crowdin-logo1-small.png" height="30">](https://crowdin.com/)

> CrowdIn lets us translate the documentation.

Reviewer: ```suggestion
Crowdin lets us translate the documentation.
```
